src/browser/starter.js

```diff
@@ -39,6 +39,11 @@
  * - `filesystem Object` (No 9p filesystem) - A 9p filesystem, see
  *   [filesystem.md](filesystem.md).
  *
+ * - `extra_images Object` (No extra images) - An object whose keys
+     are extra image names and whose values are images to load (see
+     below).  Names used by other images described above are
+     disallowed.
```

Comment:
I'm not too fond of this new parameter. It adds more things to already overly complicated constructor, and doesn't serve much use besides the test (or am I missing something?).

I'd suggest removing it, and in the test reading the file with `fs.readFileSync` and wrapping it in `v86util.SyncBuffer` (after all it's only 1.4 MB). For larger files (cdroms, eventually), we can use `v86util.AsyncXHRBuffer`.

Eventually, I plan to refactor the buffer management, so that `insert_fda` takes the same object as the `fda` property in the constructor, but let's leave that out of this PR.

Reply:
Ok, that makes sense. I was hoping to use it to e.g. set up v86 with a full suite of install floppies for some software, but I can find a way to load up images in the test suite on the fly and/or implement the idea of loading a buffer description in insert_fda, or at least introducing an async utility function on starter that takes a buffer desc and yields a buffer.